src/Sulu/Bundle/RouteBundle/Tests/Unit/Generator/SymfonyExpressionTokenProviderTest.php

```diff
@@ -55,6 +55,8 @@ public function testResolveNotExists()
     {
         $this->expectException(CannotEvaluateTokenException::class);
         $translator = $this->prophesize(Translator::class);
+        $translator->getLocale()->willReturn('en');
+        $translator->setLocale('en')->shouldBeCalled();
```

Comment:
Why did that change?

Reply:
The translator does not longer accept `null` as locale and did fail. Related to this changes #5137